pkgs/applications/altcoins/steem/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv, fetchFromGitHub
+, cmake, pkgconfig, autoconf, automake, libtool
+, boost, openssl, ncurses, readline
+, perl, doxygen, python3
+, zlib, bzip2, snappy # not currently in use; aren't truly needed by lib?
+}:
+let
+
+  version = "0.20.6";
+
+  # These values are embedded into the respective binaries, but don't appear to be crucial.
+  rev = "5d0a4327a22160eea8f1ffe5a7bad7c5ed813ac8";
+  revDate = "Wed Oct 24 10:51:32 2018";
+  sha256 = "1ylhp74myxy2ixlxrj8pfm0vfby15hzzr85ymwmlc0vba4si6in0";
+  fcRev = "EmbeddedInSteemRepo";
+  fcRevDate = revDate;
+
+  python3WithJinja = python3.pkgs.buildPythonApplication {
```

Comment:
This confuses me.

How about `python3Env = python3.withPackages (ps: [ps.jinja2])`
And use that as a `buildInputs`?
##### Documented somewhere at https://nixos.org/nixpkgs/manual/#building-packages-and-applications

Reply:
I wasn't aware of that facility, thanks.